news/_posts/2019-07-04-1.6.1.x-what-s-next.md

```diff
@@ -0,0 +1,88 @@
+
+
+## What does it mean?
+
+Put it simply, from the 1st of July 2019, the maintenance of PrestaShop 1.6.1.x is now being performed exclusively by volunteers. If a critical issue is discovered (we think mainly of security vulnerabilities), this team of volunteers will look for solutions from the community, work on it through a process of code review and extensive testing, in order to release a patch version to solve this critical issue.
+
+Those patch versions will not be officially issued by PrestaShop nor available on prestashop.com, instead, they will be available on a dedicated GitHub repository:
+[https://github.com/PrestaShop/PrestaShop-1.6](https://github.com/PrestaShop/PrestaShop-1.6)
+
+On this repository, it will be possible to submit pull requests in order to fix critical issues to be shipped in a patch version for PrestaShop 1.6.1.x.
+
+
+## Scope of the maintenance
+
+The volunteers and the PrestaShop company have agreed on the following objectives:
+
+Fix critical issues (like the recent spam attack on account creation form) or security issues
+Eventually, if this is considered possible, allow PrestaShop 1.6 to be run with PHP 7.2
+
+Consequently, non critical issues and pull requests for the 1.6 version which are not related to one of those objectives will not be merged and will be archived. This will help the new maintainers stay focused on critical patch versions. 
+
+If you find a security vulnerability in the software, the process stays the same:
+
+- Don’t speak about it publicly immediately
+- Instead, please send an email to security@prestashop.com with details about the issue, how to reproduce it, and any other interesting details
+- You can of course submit a patch proposal in the email
+- When the patch is approved and the release available, you can speak about it
+
+If you want to participate in making the 1.6.1.x work with PHP 7.2:
+
+- Please [open a pull request on the dedicated repository](https://github.com/PrestaShop/PrestaShop-1.6/pulls)
+- It will be reviewed by the maintainers, of course they might ask you more information and modifications in order to approve
+- The aim is to write good code and use best practices, and avoid “sticking-plaster”
+
+Similarly to the main open source project repository, each pull request will have to be approved by two maintainers before being merged, in order to be sure that it is well written and that it is not introducing new problems.
+
+When a patch version has been tested and approved by the maintainers, a build will be released on GitHub with the help of developers from the PrestaShop company. 
+
+
+## The volunteers
+
+A short presentation of the volunteers:
+
+- Olivier Le Corre, who started the discussion about this idea at the PS-Connect Lyon this Spring, a freelancer and long time active member of the ecosystem
+- Maxime Varinard, from the software editor Vaisonet, a specialist on synchronisation of PrestaShop with other information systems
```

Comment:
synchronization in english ;-)

Reply:
US and UK English differences. let's add a z ;-)